internal/conformance/config.yaml

```diff
@@ -0,0 +1,21 @@
+features:
+  versions:
+    - HTTP_VERSION_1
+    - HTTP_VERSION_2
+    - HTTP_VERSION_3
+  protocols:
+    - PROTOCOL_CONNECT
+    - PROTOCOL_GRPC
+    - PROTOCOL_GRPC_WEB
+  codecs:
+    - CODEC_PROTO
+    - CODEC_JSON
+    - CODEC_TEXT
```

Comment:
Nothing to do here, just an idle comment on the not-great content-type we end up with. It's nice that we can test this in the conformance suite, though!

Reply:
Yep, it's true. I mainly added to the conformance tests because it is a codec that is supported by default by the grpc-web JS stuff (since they can't support JSON because they're using a protobuf runtime w/out JSON support, that is the codec to use for human-readable payloads).

Similarly, all the compression algorithms listed are ones that are actually referenced in the Connect wire protocol spec (br, zstd) or the gRPC HTTP/2 protocol spec (deflate, snappy). So if someone implements them, it seemed reasonable to be able to check them.

Reply:
No, the text encoding is not the same as the base64-encoding of binary stuff -- grpc-web-text does not produce human readable messages in the network inspector.

I can't seem to find the support for the text codec anymore. Maybe it was removed. At FullStory, we investigated using grpc-web circa 2017/2018. That was before there was an official one, so we were looking at the TS stuff in [improbable-eng/grpc-web](https://github.com/improbable-eng/grpc-web). We experimented with using the text codec. (I remember having to configure our gRPC servers to support it, since it is not a supported codec out-of-the-box in grpc-go. I also remember being very confused and frustrated that there was no support in protobuf-js for JSON.)

So it may not be present anymore, in either the official implementation or the improbable-eng one. But I had added it at as a supported codec in the conformance tests from my memory of that. We can always remove it if we think no one would ever use it.

Reply:
I can see how the text format would have looked like a good option back then. Doesn't seem to hurt to test it. It is definitely not used anywhere I know with gRPC-Web.